### ISSUE TITLE
The results measured by the deep3d_pytorch validation set have the mean error 1.87mm and median error 1.78mm

### ISSUE THREAD

**Issue by DougDong666** [soubhiksanyal/now_evaluation#27]
the results in now challenge measured by the deep3d_pytorch test set only have 1.41mm mean error and 1.11mm median error. Is this a normal situation or is there something wrong with my method? I use the pretrained weight of deep3d_pytorch to generate the mesh with no expressions.

**Comment by TimoBolkart** [comment]
While we have seen validation errors to be 0.1-0.2 mm higher than the actual test set error, a difference of 0.76 mm seems too high. The authors of deep3d_pytorch have provided the test set meshes to us, so I don't know exactly how they obtained these. 
Some methods output only a tight face crop of BFM meshes, which results in a higher error in the forehead and close to the face boundary. Instead, providing the uncropped BFM meshes reduces the error. But I don't know if this is what deep3d_pytorch people did.

**Comment by DougDong666** [comment]
The deep3d_pytorch use the cropped mesh, which does not contain the jaw. And this is a big region which makes the mean error and median error so big. when other people test the deep3d_pytorch, I suggest you can only get First 80 dimension output parameter, which is the id parameter, to generate the non-cropped mesh.

**Comment by DougDong666** [comment]
However, this time I find some meshes are not aligned，IMG_201 I got 33.7mm mean error and some other images I got 8mm mean error. I think it doesn't converge
![image](https://user-images.githubusercontent.com/88541152/212028068-6d63d51d-650a-4f0a-b4bc-d73dabc590cd.png)

**Comment by DougDong666** [comment]
the mean error of non-cropped mesh is 1.39mm and the median error of it is 1.27mm

**Comment by Oliver-YX** [comment]
> The deep3d_pytorch use the cropped mesh, which does not contain the jaw. And this is a big region which makes the mean error and median error so big. when other people test the deep3d_pytorch, I suggest you can only get First 80 dimension output parameter, which is the id parameter, to generate the non-cropped mesh.

I wonder how to only get the first 80 dimension output parameters when we test the deep3d_pytorch. I meet the same problem and the result is 1.38mm mean error and 2.02mm median error. Thanks for your answer
